src/tools/asyncstacktraces/ShadowStackEntry.java

```diff
@@ -0,0 +1,62 @@
+package tools.asyncstacktraces;
+
+import com.oracle.truffle.api.source.SourceSection;
+
+import som.Output;
+import som.interpreter.actors.Actor;
+import som.interpreter.actors.Actor.ActorProcessingThread;
+import som.interpreter.actors.EventualMessage;
+import som.interpreter.nodes.ExpressionNode;
+import tools.SourceCoordinate;
+
+
+public abstract class ShadowStackEntry {
+
+  protected final ShadowStackEntry previousEntry;
+  protected final ExpressionNode   expression;
+  public static long               numberOfAllocations = 0;
```

Comment:
Hum, that is an analysis only variable... Not sure it makes sense to look into this.

Reply:
Unclear why you initialize `numberOfAllocations` and `ALLOCATION_COUNT ` explicitly. `0` and `false` are the default values.

Also, for style, it would be nice to have a free line before `numberOfAllocations` to group the two separate things (instance vs class state) visually.

Reply:
I will remove the initialization to zero, that will keep codacy happy. The other one I keep because I modify it frequently... I can make it a VM setting
 but I do not want to add a crazy number of settings.

Reply:
What's the problem with making it a proper `VmSetting`? Makes things much easier to use, no?
Also, you don't need to add it to the launcher. A simple `./som -Dsom.someSetting=true ...` will work fine.